assets/js/idx-leads.js

```diff
@@ -199,7 +199,8 @@ jQuery(document).ready(function($) {
 			},
 			success: function( result ) {
 				if( $.isNumeric( result ) ) {
-					window.location.href = leadurl + result;
+					alert('Lead Successfully Added');
+					window.location.reload(true);
```

Comment:
I assume this `alert` was just for testing. I noticed when working with the omnibar, just setting `window.location = 'newurl'` reloaded the page. Not saying it needs to be removed, but I am curious if the reload true makes a difference in the functionality.

Reply:
I left the alert in there as some kind of visual queue that the lead was added.

Reply:
Without the alert the page just reloads and empties the field

Reply:
I removed the alert after seeing that there isn't an alert for other actions.